src/main/java/util/csvparsing/CsvMonumentConverter.java

```diff
@@ -104,4 +361,27 @@ private static String removeBeginningAndEndingQuotes(String string) {
 
         return string;
     }
+
+    /**
+     * Static method to add the beginning and ending quotes to a specified String
+     * Does nothing if there are already beginning and ending quotes
+     * @param string - the String to add the quotes to
+     * @return String - the updated String, with beginning and ending quotes
+     */
+    private static String addBeginningAndEndingQuotes(String string) {
+        if (string.startsWith("\"") && string.endsWith("\"")) {
+            return string;
+        }
+
+        return "\"" + string + "\"";
+    }
+
+    /**
+     * Static method to get the header row for CSV file outputs
+     * @return String - the CSV header row
+     */
+    public static String getHeaderRow() {
+        return "ID,Title,Artist,Date,Material,Latitude,Longitude,City,State,Address,Inscription,Description," +
+                "Tags,Contributors,References";
```

Comment:
This was just the order I decided on for the purposes of this quick task. When we come back to this as part of the export to CSV requirement we can determine a new order.